java/vector/src/test/java/org/apache/arrow/vector/TestFixedSizeListVector.java

```diff
@@ -243,6 +244,29 @@ public void testTransferPair() {
     }
   }
 
+  @Test
+  public void testTransferEmptyVector() throws Exception {
+    // #43320
```

Comment:
Need this?

Reply:
It is useful to record which tests are regression tests for certain issues.